### PR TITLE
Updated the property step and filter request payload

### PR DESCRIPTION
**Pull request by indraraj**
- Wired the property page forms

- Removed the hardcoded value for filter request.

### CODE REVIEW

File: ui/packages/ui/src/app/pages/createConnector/connectorSteps/ConfigureConnectorTypeForm.tsx [hunk 1]
```diff
@@ -56,58 +78,85 @@ export const ConfigureConnectorTypeForm: React.FunctionComponent<IConfigureConne
...
+    _.union(
+      basicPropertyDefinitions,
+      advancedGeneralPropertyDefinitions,
+      advancedReplicationPropertyDefinitions,
+      advancedPublicationPropertyDefinitions
+    )
+  );
+
+  const handleSubmit = (valueMap: Map<string, string>) => {
+    // Validate the basic properties
```
Comment (mdrillin) [inline ui/packages/ui/src/app/pages/createConnector/connectorSteps/ConfigureConnectorTypeForm.tsx:133]: I think the basic and advanced properties could be combined - would result in just a single call to the backend.  wdyt?

Comment (indraraj) [reply ui/packages/ui/src/app/pages/createConnector/connectorSteps/ConfigureConnectorTypeForm.tsx:133]: I separated as I thought we might handle(logically) the "valid"/"invalid" response differently to basic and advance property.

Comment (mdrillin) [reply ui/packages/ui/src/app/pages/createConnector/connectorSteps/ConfigureConnectorTypeForm.tsx:133]: I think all properties should be validated at once, but right, we may want to handle the response differently depending on whether the problem property is basic or advanced.

Comment (indraraj) [reply ui/packages/ui/src/app/pages/createConnector/connectorSteps/ConfigureConnectorTypeForm.tsx:133]: yeah, we can check it with Gunnar & Team if they want the user to move ahead if only basic props are valid

**Review by mdrillin — APPROVED** [review]
Looks good

**Pull request merged**